docs/system_overview/configuration.md

```diff
@@ -69,6 +81,28 @@ This file defines the behavior of the server Agent that answers individual reque
         // The amount of time a temporary password remains valid.
         "maximum_temporary_password_lifetime_in_seconds": 1000,
 
+        // (Optional)
+        // The number of seconds between runs of the CRON task which runs the delay server migration.
+        "migrate_delay_server_sleep_time_in_seconds": 5,
+
+        // (Optional)
+        // Sets the timeout for receive operations via ZeroMQ's setsockopt API for delay server migration communications.
+        // The following values are considered for controlling zmq_recv:
+        //  - 0:    Returns immediately, with a EAGAIN error if there is no message to receive.
```

Comment:
Should `a EAGAIN` be `an EAGAIN`? I'm not sure which is correct.

_Same for the one just below._

Reply:
Yep, I think it should. I copied this straight out of their docs and made some tweaks so the wording made sense. This will be tweaked as well!